### PR TITLE
add 2D lgad simulation

### PR DESCRIPTION
**Pull request by yangtaogit**

### CODE REVIEW

File: raser/calcurrent.py [hunk 1]
```diff
@@ -12,7 +12,13 @@
 import ROOT
 import math
 import sys
+from array import array
+from model import Mobility
+from model import Avalanche
 
+""" Global Constant """
```
Comment (xshi) [inline raser/calcurrent.py:19]: Can we move the global constant into another constant.py and import them separately? i.e.

import constant 
use them as: constant.e0

File: raser/model.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
```
Comment (xshi) [inline raser/model.py:1]: if this "model.py" is only use as "import" by other file, not recommend to make it executable.

**Comment by xshi** [comment]
Please remove all the commented debugging lines (print) to ease the review process. 

Also, there should be no file needed to be executable.

**Comment by xshi** [comment]
@tanyh2018  if you've done your review, please just type 👍

**Comment by tanyh2018** [comment]
+1

**Pull request merged**